numba_dpex/tests/debugging/test_breakpoints.py

```diff
@@ -98,11 +97,16 @@ def test_breakpoint_common(app, breakpoint, script):
     setup_breakpoint(app, breakpoint, script=script)
 
 
+@pytest.mark.xfail  # TODO: https://github.com/IntelPython/numba-dpex/issues/1221
```

Comment:
Should we run this benchmark both for `numba` and `numba_dpex` and xfail only for dpex?

Reply:
`simple_sum.py` only has implementation for numba_dpex and not for numba.